db/upgrade.php

```diff
@@ -164,5 +164,28 @@ function xmldb_accredible_upgrade($oldversion=0) {
         upgrade_mod_savepoint(true, 2022060900, 'accredible');
     }
 
+    if ($oldversion < 2024041600) {
```

Comment:
Can this number be a constant?

Reply:
I'm not sure. The docs point to it being as is.